@navikt/core/react/src/util/i18n/locales/en.ts

```diff
@@ -0,0 +1,27 @@
+import type nb from "./nb";
+
+export default {
+  FileUpload: {
+    dropzone: {
+      button: "Select file",
+      buttonMultiple: "Select files",
+      dragAndDrop: "Drag and drop file here",
+      dragAndDropMultiple: "Drag and drop files here",
```

Comment:
Would just having "Drop" be enough? Is this the case for nb and nn to then?
```suggestion
      dragAndDrop: "Drop file here",
      dragAndDropMultiple: "Drop files here",
```

Reply:
I'm really not sure. Checked what other design systems do:
- Shopify: Button labeled "Add files"
- Adobe: "Drag and drop your file"
- Grafana: "Upload file"
- iTwinUI: "Choose a file or drag & drop it here."
- Zendesk Garden: "Drag files here or click to upload"
- Carbon: "Drag and drop files here or click to upload"
- Amplify: "Drag images here"
- Salt Design System: "Drop files here or browse files"
- Evergreen: "Browse or drag a file here"
- Ant: "Click or drag file to this area to upload"

Summary:
- Drag + drop: Adobe, iTwinUI, Carbon (3)
- "Drag" only: Zendesk, Amplify, Evergreen, Ant (4)
- "Drop" only: Salt (1)

Based on this limited survey, it seems like just "drop" is the least popular. In Norwegian, I personally think that "dra filer hit" sounds better than "slipp filer her". So if we are only going to use one of the words, I would vote for "drag".

Visually, I think it actually looks better with the current text. The short version kinda makes the content look too narrow compared to the container:
![image](https://github.com/user-attachments/assets/002c58ca-cbad-4460-961f-f1b643e09399)

Anyways, I think we should do the change in a separate PR if we decide to do it.

Reply:
This is really thorough! 🚀  (I also like the second one for the same reason, looks more balanced)